lms/envs/common.py

```diff
@@ -408,6 +408,9 @@
 
     # Enable LTI Provider feature.
     'ENABLE_LTI_PROVIDER': False,
+
+    # Enable automatic status emails for ICRV attempts
+    'ENABLE_ICRV_STATUS_EMAILS': False,
```

Comment:
I intend to add Waffle support as part of #10096. However, given that work is unscheduled, we should use this opportunity to add Waffle now. That's the only way we'll break the bad habit.